include/kindyn/robot.hpp

```diff
@@ -126,6 +126,29 @@ namespace cardsflow {
                 ROS_WARN_STREAM_THROTTLE(1, "writing virtual, "
                         "you probably forgot to implement your own write function?!");
             };
+
+            /**
+             * Setter functions for gymFunctions class.
+             */
+            void setMotorState(int index1, int index2, double value);
+
+            void setJointState(int index1, int index2, double value);
+
+            void setIntegrationTime(double dt);
+
+            /**
+             * Getter functions for gymFunctions class
+             */
+            virtual double getLimit(int index1, int index2) {};
+
+            virtual vector<double> getLimitVector(int index) {};
+
+            virtual double getMax(int index) {};
+
+            virtual double getMin(int index) {};
+
+            virtual bool getExternalRobotState() {};
```

Comment:
we request an external robot state (what is this?) and get a boolean. This is very confusing. What is this and can we rename it? :)